dev-lang/jimtcl/files/jimtcl-0.78-jimsh0-remove-redundant-JimRestoreEnv.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/autosetup/jimsh0.c b/autosetup/jimsh0.c
```

Comment:
I'm afraid this doesn't help at all.

Reply:
Looks like I've just deleted MINGW specific implementation.

I will look for workaround if no obvious solution is found.

Reply:
Hmm, FWICS upstream is still alive and kicking, so maybe try adding a new version (or a git snapshot if upstream doesn't release).

Reply:
Good idea. I will look for changes on a week. May be I'll find something.

If someone here needs a fast workaround then let me know. We can just bypassing bootstrapping by forcing `tclsh` usage.